docker-compose.yml

```diff
@@ -588,6 +588,8 @@ services:
       - _APP_DOMAIN
       - _APP_OPTIONS_FORCE_HTTPS
       - _APP_DATABASE_SHARED_TABLES
+      - _APP_DOMAIN
```

Comment:
Those two variables are in rows [588-589](https://github.com/appwrite/appwrite/pull/8722/files#diff-e45e45baeda1c1e73482975a664062aa56f20c03dd9d64a827aba57775bed0d3R588-R589)

Reply:
Oops! Updated!